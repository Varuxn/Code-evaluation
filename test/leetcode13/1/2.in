100 400
49 45
24 33
3 81
12 44
55 6
97 89
78 82
39 59
11 20
26 61
35 98
98 29
36 78
99 68
55 40
53 65
43 3
31 39
91 63
19 17
74 78
3 9
25 85
66 16
15 76
67 83
63 17
79 93
93 60
69 42
63 58
2 7
88 34
60 14
2 35
9 10
34 68
67 88
52 92
88 6
17 69
27 31
55 95
21 9
68 44
43 65
0 48
82 74
26 97
50 43
8 58
9 43
16 46
92 65
84 80
57 66
35 7
85 48
9 27
51 18
8 6
58 83
6 29
90 22
15 37
4 63
53 0
49 16
6 23
3 17
95 67
75 61
6 48
4 28
3 36
78 80
67 12
53 68
2 43
34 40
75 77
69 98
58 97
5 18
0 93
6 8
20 10
28 32
30 34
91 84
11 52
96 28
93 29
94 65
3 73
1 90
1 4
15 50
18 33
2 63
12 30
19 28
16 26
40 29
31 33
72 26
71 22
10 15
54 26
51 59
34 25
45 17
40 75
11 5
16 56
70 97
37 43
76 67
9 49
96 54
44 84
17 14
86 99
10 26
38 45
71 99
44 67
9 82
9 42
25 7
15 21
93 87
98 60
8 65
9 98
37 28
12 41
78 2
91 34
88 61
75 71
9 50
72 90
49 64
33 0
80 79
36 75
24 4
27 71
78 85
15 49
91 25
85 58
82 29
33 45
92 78
80 74
1 3
68 71
82 84
50 75
41 81
55 87
52 76
28 7
9 72
1 72
89 7
1 2
29 13
16 66
89 34
43 96
24 88
7 94
70 74
50 11
66 90
28 76
65 44
18 66
66 17
30 60
79 81
89 59
29 58
10 80
33 66
19 39
6 38
32 85
44 96
57 28
19 41
3 69
60 85
70 17
91 8
9 4
63 96
8 79
93 88
74 24
60 2
89 58
38 98
47 1
67 99
3 93
58 77
33 53
28 96
24 57
27 82
76 71
36 99
83 99
88 17
0 24
0 91
51 44
85 68
25 81
42 42
68 67
1 69
22 91
91 77
56 57
10 29
78 19
70 25
1 31
33 99
62 19
25 89
0 55
29 36
24 31
63 70
39 83
96 61
18 20
61 96
5 16
72 88
41 28
13 42
6 56
79 1
45 89
19 77
97 38
0 6
59 77
88 74
30 69
16 32
13 14
50 99
5 42
42 68
94 63
74 84
10 23
13 39
0 1
20 89
61 76
61 43
16 43
94 95
36 62
10 73
3 41
26 82
70 38
7 11
18 38
38 87
71 57
34 39
38 53
35 30
66 92
94 16
55 43
77 84
23 51
3 37
68 70
3 38
68 55
99 23
15 34
33 44
66 67
6 7
41 68
89 89
48 26
43 37
47 67
58 1
23 6
41 63
27 81
5 97
40 10
75 85
5 28
23 68
98 69
61 6
59 57
54 83
14 78
31 84
16 65
41 30
49 97
65 39
9 18
23 41
5 13
9 46
69 55
0 5
31 62
34 79
45 11
30 89
20 57
70 75
46 89
42 21
56 81
26 92
64 77
8 94
48 58
65 78
4 12
35 64
70 2
83 73
47 99
37 84
2 12
51 96
38 58
49 61
89 20
9 47
58 63
43 53
14 89
75 68
39 53
9 78
95 22
89 48
50 79
41 64
39 40
3 16
13 22
37 90
71 35
82 2
64 37
39 46
3 54
30 86
56 49
27 79
82 55
89 72
49 8
56 26
69 35
21 25
14 53
2 28
52 47
13 91
37 60
13 81
13 24
10 19
64 10
17 87
91 42
13 68
35 80
36 19
54 51
30 52
44 13
56 39
