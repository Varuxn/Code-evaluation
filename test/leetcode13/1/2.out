73 81 87 